src/cbmc/bmc.h

```diff
@@ -15,6 +15,7 @@ Author: Daniel Kroening, kroening@kroening.com
 #include <list>
 #include <map>
 
+#include <util/ui_message.h>
 #include <util/options.h>
```

Comment:
As you do lexicographic sorting of includes elsewhere: do this here as well?

Reply:
Fixed.